db/sql/cassandra/query.go

```diff
@@ -38,7 +38,7 @@ func PutExpToString(whereCondition sql.Expression, entity interface{}) (sqlStr s
 		return "", nil, err
 	}
 
-	bindings = structs.ListExportedFieldsPtrs(entity, cqlExported)
+	bindings = structs.ListExportedFieldsPtrs(entity, true, cqlExported)
```

Comment:
please change the:

 structs.ListExportedFieldsPtrs(entity, true, cqlExported)

to:
  structs.ListExportedFieldsPtrs(entity, cqlExported, filterOutPK)